src/webui/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fluentui/react": "^7.135.0",
```

Comment:
Please sort the dependencies alphabetically. (It should be automatic if you had add them with `yarn add`)

Reply:
`yarn add @fluentui/react` and it is in this line.

Reply:
@ultmaster by test, @fluentui/react will in "devdependencies"